src/go/intelligence/README.md

```diff
@@ -132,7 +127,7 @@ The given config file format should be followed. The config file consists of mai
     "verbose": 0,
     "dryRun": false,
     "testing": {
-      "testfile": "~/CMSMonitoring/src/go/intelligence/test/test_cases.json",
+      "testfile": "~/CMSMonitoring/src/go/intelligence/int_test/test_cases.json",
```

Comment:
I said many times don't use relative path. Here it refers to HOME where user may not be comfortable to keep CMSMonitoring. Instead, either use environment variable, e.g. `$CMSMONITORING`, or full path, e.g. /tmp/test_cases.json, or just file name. Based on your choice you should adjust test script to setup this area or do proper action, e.g. cd /tmp; git clone ...;, etc.

Reply:
fixed this issue.